oonidata/db/create_tables.py

```diff
@@ -53,50 +55,69 @@ def typing_to_clickhouse(t: Any) -> str:
     if t == Outcome:
         return "String"
 
+    if t == dict[str, str]:
+        return "Map(String, String)"
+
     raise Exception(f"Unhandled type {t}")
 
 
-def create_query_for_observation(obs_class: Type[Observation]) -> str:
+def create_query_for_observation(obs_class: Type[Observation]) -> Tuple[str, str]:
     columns = []
     for f in fields(obs_class):
         type_str = typing_to_clickhouse(f.type)
-        columns.append(f"     `{f.name}` {type_str}")
+        columns.append(f"     {f.name} {type_str}")
 
     columns_str = ",\n".join(columns)
 
-    return f"""
+    return (
+        f"""
     CREATE TABLE {obs_class.__table_name__} (
 {columns_str}
     )
     ENGINE = ReplacingMergeTree
     ORDER BY (timestamp, observation_id, measurement_uid)
     SETTINGS index_granularity = 8192;
-    """
+    """,
+        obs_class.__table_name__,
+    )
 
-def create_query_for_verdict() -> str:
+
+def create_query_for_verdict() -> Tuple[str, str]:
     columns = []
     for f in fields(Verdict):
         type_str = typing_to_clickhouse(f.type)
-        columns.append(f"     `{f.name}` {type_str}")
+        columns.append(f"     {f.name} {type_str}")
 
     columns_str = ",\n".join(columns)
 
-    return f"""
+    return (
+        f"""
     CREATE TABLE verdict (
 {columns_str}
     )
     ENGINE = ReplacingMergeTree
     ORDER BY (timestamp, verdict_id, measurement_uid)
     SETTINGS index_granularity = 8192;
-    """
+    """,
+        "verdict",
+    )
+
 
 def main():
-    print(create_query_for_observation(DNSObservation))
-    print(create_query_for_observation(TCPObservation))
-    print(create_query_for_observation(TLSObservation))
-    print(create_query_for_observation(HTTPObservation))
+    create_queries = [
+        create_query_for_observation(DNSObservation),
+        create_query_for_observation(TCPObservation),
+        create_query_for_observation(TLSObservation),
+        create_query_for_observation(HTTPObservation),
+        create_query_for_observation(NettestObservation),
+        create_query_for_verdict(),
+    ]
+    for query, table_name in create_queries:
+        print(f"clickhouse-client -q 'DROP TABLE {table_name}';")
+        print("cat <<EOF | clickhouse-client -nm")
+        print(query)
+        print("EOF")
```

Comment:
I am confused by this code 😕 

Is this a way to emit on the stdout what clickhouse should do? It seems you're generating a script?!

Reply:
Basically, now while I am developing this thing, since I don't really care about having the data in the tables preserved, whenever I make changes to the schema I just drop the old tables and re-create them.

By running the `create_tables.py` script I get in stdout the commands I need to copy paste into the shell to do that.